### ISSUE TITLE
undefined reference to

### ISSUE THREAD

**Issue by embbo** [open62541/open62541#4489]
## Description
I just compiled the latest files .. cmake and make .. and obtain open62541.h/62541.c ...but i got these errors .. I am using eclipse ..but old files works fine ..
with this function:
 UA_ServerConfig_setDefaultWithSecurityPolicies( uaServerConfig,
			 portNumber,
			 &certificate,     &privateKey,
			 0,        0,
			 0,        0,
			 0,        0
			 );

/../Src/open62541.c:70232: undefined reference to `UA_SecurityPolicy_Basic128Rsa15'
/../Src/open62541.c:70266: undefined reference to `UA_SecurityPolicy_Basic256'
/../Src/open62541.c:70300: undefined reference to `UA_SecurityPolicy_Basic256Sha256'
/../Src/open62541.c:70334: undefined reference to `UA_SecurityPolicy_Aes128Sha256RsaOaep'
/../Src/open62541.c:70417: undefined reference to `UA_CertificateVerification_Trustlist'

and these are declared in open62541.h

## Background Information / Reproduction Steps

Used CMake options:

```bash
cmake -DUA_NAMESPACE_ZERO=<YOUR_OPTION> <ANY_OTHER_OPTIONS> ..
```

## Checklist

Please provide the following information:

 - [x] open62541 Version (release number or git tag):
 - [x] Other OPC UA SDKs used (client or server): 
 - [ ] Operating system:
 - [ ] Logs (with `UA_LOGLEVEL` set as low as necessary) attached
 - [ ] Wireshark network dump attached
 - [ ] Self-contained code example attached
 - [x] Critical issue

**Comment by jackybek** [comment]
You need to Enable the CCmake options relating to encryption.

**Comment by embbo** [comment]
@jackybek  i did thats why it is  calling that funtion...

**Comment by jackybek** [comment]
Can you list the ccmake options that are enabled ? 
Did you turn on almagamation option ?
Did you turn on built shared lib option ?
In my case, if I use almagamation and don’t enable shared lib, I don’t encounter missing references, but the moment I turn off shared lib, I get funny issues similar to yours.

**Comment by embbo** [comment]
@jackybek yes i did , actually I make other old version it works fine ... but in this we both have same issue ... i don't know why

**Comment by embbo** [comment]
@jackybek Yes without encryption , it works fines but with library ...it gives that issues

**Comment by andreasebner** [comment]
As there's been no further activity and the projects code has changed a lot since then, I'll close this issue.
